sensors/jira_sensor_for_apiv2.py

```diff
@@ -105,9 +105,9 @@ def _dispatch_issues_trigger(self, issue):
         payload = {}
         payload['project'] = self._project
         payload['id'] = issue.id
-        payload['expand'] = issue.raw['expand']
+        payload['expand'] = issue.raw.get('expand', None)
         payload['issue_key'] = issue.key
         payload['issue_url'] = issue.self
         payload['issue_browse_url'] = self._jira_url + '/browse/' + issue.key
-        payload['fields'] = issue.raw['fields']
+        payload['fields'] = issue.raw.get('fields', None)
```

Comment:
You'd better match the type of default value with [the type of payoad_schema](https://github.com/StackStorm-Exchange/stackstorm-jira/pull/18/files#diff-930ea5895f672422f484bd7929e61dffR26).